### ISSUE TITLE
Unmatched parentheses in license expressions ABNF

### ISSUE THREAD

**Issue by gstroz** [spdx/spdx-spec#504]
New to ABNF and grammar rules but I believe there's an unmatched bracket in the notation. Specifically line 29 in https://github.com/spdx/spdx-spec/blob/development/v2.2.1/chapters/SPDX-license-expressions.md#L29 has an extra `)` after the second `compound-expression` which is unmatched. 

```
compound-expression = 1*1(simple-expression /


simple-expression "WITH" license-exception-id /

  compound-expression "AND" compound-expression /

  compound-expression "OR" compound-expression ) /    <--- bracket in this line

  "(" compound-expression ")" )
```

**Comment by zvr** [comment]
Actually, one can simplify the whole expression by deleting the initial `1*1` and all the grouping parentheses.

**Comment by goneall** [comment]
Resolved with #505
